lib/matplotlib/ticker.py

```diff
@@ -2634,7 +2634,15 @@ def __init__(self, n=None):
 
         If *n* is omitted or None, it will be set to 5 or 4.
         """
-        self.ndivs = n
+        if n == None:
+            if Locator.axis.__name__ == 'xaxis':
```

Comment:
At that point `self.axis` is not set yet, but it should be in `__call__`.

Reply:
:slightly_frowning_face: That means we can only to the default resolution in `__call__` and not in `__init__`. That's a bit unusual, but there's probably no way around; and hopefully nobody will stumble over it.